src/forwardBeamPipeBrazil.cpp

```diff
@@ -0,0 +1,570 @@
+// SPDX-License-Identifier: LGPL-3.0-or-later
+// Copyright (C) 2024 Alex Jentsch
+
+//==========================================================================
+//
+//      <detector name ="DetName" type="Beampipe" >
+//      <layer id="#(int)" inner_r="#(double)" outer_z="#(double)" >
+//      <slice material="string" thickness="#(double)" >
+//      </layer>
+//      </detector>
+//==========================================================================
+#include "DD4hep/DetFactoryHelper.h"
+#include "DD4hep/Printout.h"
+#include "TMath.h"
+#include <XML/Helper.h>
+
+using namespace std;
+using namespace dd4hep;
+
+/** \addtogroup beamline Beamline Instrumentation
+ */
+
+/** \addtogroup IRChamber Interaction Region Vacuum Chamber.
+ * \brief Type: **IRChamber**.
+ * \ingroup beamline
+ *
+ *
+ * \code
+ *   <detector>
+ *   </detector>
+ * \endcode
+ *
+ */
```

Comment:
Does not apply (or lacks specificity).

Reply:
I really didn't even know what this was.